components/automate-ui/src/app/entities/policies/policy.effects.ts

```diff
@@ -97,7 +97,7 @@ export class PolicyEffects {
       const removeStr = removed.length === 1 ? removed[0].displayName : `${removed.length} members`;
       return new CreateNotification({
           type: Type.info,
-          message: `Removed ${removeStr}.`
+          message: `Removed member(s) ${removeStr}.`
```

Comment:
Is ` - ` needed before `${removeStr}` ?

Reply:
OK, the answer here is "no". This is still the old format, the difference is the word "Successfully" is not present here. Jon from UX is well aware of the inconsistencies--I spoke to him about it a couple days ago. Apparently, new code we want to go in with "Successfully <verb> <noun> - <id>" but I purposely did not change this one because it is "old" code.